CompilerSource/parsing/pretty_printer.cpp

```diff
@@ -546,14 +572,18 @@ bool AST::CppPrettyPrinter::VisitNewExpression(AST::NewExpression &node) {
 }
 
 bool AST::CppPrettyPrinter::VisitDeclarationStatement(AST::DeclarationStatement &node) {
-  if (node.storage_class == DeclarationStatement::StorageClass::GLOBAL ||
-      node.storage_class == DeclarationStatement::StorageClass::LOCAL) {
+  bool is_global = node.storage_class == DeclarationStatement::StorageClass::GLOBAL;
+  bool is_local = node.storage_class == DeclarationStatement::StorageClass::LOCAL;
+  if (is_global || is_local) {
     bool printed = false;
     for (std::size_t i = 0; i < node.declarations.size(); i++) {
       if (node.declarations[i].init) {
         if (printed) print(", ");
         std::string name = node.declarations[i].declarator->decl.name.content;
-        print(name + " = ");
+        if (is_global)
+          print("enigma::varaccess_" + name + "(int(global)) =");
```

Comment:
Nit: string concatenation is more expensive than repeated stream-put operations. The system will run a little faster if we just `print(x); print(y); print(z);` instead of `print(x + y + z);`.